README.md

```diff
@@ -1,27 +1,25 @@
-# bpmn-visualization
+<h1 align="center">BPMN Visualization</h1> <br>
+<p align="center">
```

Comment:
:question: did you try to center the whole introduction? I am curious to know if this improve the layout.
I can give a try at it if you want

Reply:
![image](https://user-images.githubusercontent.com/4921914/93783648-3a97aa00-fc2c-11ea-99c4-820f160c754e.png)

Reply:
I prefer this one: 
<img width="1077" alt="image" src="https://user-images.githubusercontent.com/4921914/93783976-aed24d80-fc2c-11ea-8dc8-3cc1ba0c492d.png">

Reply:
I like the second one also


I was thinking maybe for the picture instead of having Abstract Task 1 , User Task 2 .... we can have something Hacktoberfest related ? 

Hacktoberfest challenge     ->     submit 5 Pull Requests     ->     Get free Tshirt

Reply:
Yes, with the Hacktoberfest colors ;)

Reply:
But, I think it's better to do this in another PR ^^

Reply:
Ok for the 2nd option (not centered description) and for an updated BPMN diagram in another PR

Reply:
Reopen this thread as I did new experiment on this topic to center the list (`<ul style="width: 70%; margin: auto;">`)
IMHO it is clearer

![image](https://user-images.githubusercontent.com/27200110/93865184-42a42800-fcc6-11ea-9af9-1cf9008cd54a.png)

Reply:
You can commit your change 😉